docs/verify/local-testing-mode.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Qiskit Runtime local testing mode
+description: Use local testing mode to simulate quantum hardware in Qiskit Runtime
+---
+
+<span id="test-locally"></span>
+# Qiskit Runtime local testing mode
+
+Quantum simulators should be used to help develop and test programs before fine-tuning them and sending them to real quantum hardware. Local simulators can do this with good performance and efficiency.  Clifford circuits can be simulated very efficiently, and results can be verified. This is useful to give confidence in an experiment.
+
+To use local testing mode with Qiskit Runtime, specify one of the fake backends in ``qiskit_ibm_runtime.fake_provider`` or specify a Qiskit Aer backend when instantiating a primitive or a session.  For details, see the [fake_provider API reference.](/api/qiskit-ibm-runtime/fake_provider)
+
+If you currently use Qiskit (and not Qiskit Runtime), you can use the existing Qiskit Aer primitives for local testing.  See [exact and noisy simulation with Qiskit Aer primitives](../verify/simulate-with-qiskit-aer) for details. 
+
+Example:
```

Comment:
I expected the examples listed in the migration guide to also be listed here, since new users would not read this page but not the migration guide.